corehq/tabs/uitab.py

```diff
@@ -66,12 +66,9 @@ def request_path(self):
         return self._request.get_full_path()
 
     @property
-    def restrict_access_by_location(self):
-        """Is this a web user who can't access project-wide data?"""
-        return (
-            self.couch_user and self.domain
-            and not self.couch_user.has_permission(self.domain, 'access_all_locations')
-        )
+    def can_access_all_locations(self):
+        """Is this a web user who can access project-wide data?"""
+        return getattr(self._request, 'can_access_all_locations', True)
```

Comment:
i'm not sure it's good to default this setting to True, typically you'd default access to False right?